src/get-started/java-binding.md

```diff
@@ -321,31 +321,36 @@ framework.
 
 Exonum uses [Guice](https://github.com/google/guice) to describe the
 dependencies of the service components (both system-specific
-ones, for example, Exonum time service, and external ones). Each
-service should define a module describing implementations of the framework
-interfaces – `Service`, `TransactionConverter` and implementations of other
-components, if any.
+ones, for example, Exonum time service, and external ones). 
+Each service should define a Guice module describing implementations of
+the `Service` and its dependencies, if any. The modules must be `public`,
```

Comment:
How about to rewrite the sentence to something like:
Each module should meet following requirements:
* It shall extend [`AbstractServiceModule`][abstract-service-module-javadoc]
* It should be `public`
* It should be annotated with `@org.pf4j.Extension`